packages/elements/src/utils/oas/security.ts

```diff
@@ -33,7 +33,7 @@ const getOauthScopeMapper = (securityScheme: IOauth2SecurityScheme) => (flow: st
 };
 
 export function getServiceUriFromOperation(uri: string) {
-  const match = uri?.match(/(.*)\/paths/);
+  const match = uri?.match(/(.*)\/(paths|operations)/);
```

Comment:
This feels potentially brittle as we might add more URL shortcuts and cleaning in the future, but I accept that this is a bug fix and not time for tech debt. If anyone can think of anything smart to do please make an issue with ideas, but otherwise this is a good change. Thank you for adding a test!!